lib/active_record_florder/base.rb

```diff
@@ -58,6 +58,12 @@ def move(position)
         update_attribute(position_attr_name.to_sym, normalized_position)
       end
 
+      # populating all affected records
+      if move_populate_all
```

Comment:
I would suggest `return_all_affected` as configurable property - it says more what we are config by this property.